pytensor/scalar/math.py

```diff
@@ -1281,6 +1281,36 @@ def c_code(self, *args, **kwargs):
 ive = Ive(upgrade_to_float, name="ive")
 
 
+class Kv(BinaryScalarOp):
+    """Modified Bessel function of the second kind of real order v."""
+
+    nfunc_spec = ("scipy.special.kv", 2, 1)
+
+    @staticmethod
+    def st_impl(v, x):
+        return scipy.special.kv(v, x)
+
+    def impl(self, v, x):
+        return self.st_impl(v, x)
+
+    def L_op(self, inputs, outputs, output_grads):
+        v, x = inputs
+        [out] = outputs
+        [g_out] = output_grads
+        # -(v / x) * kv(v, x) - kv(v - 1, x)
```

Comment:
There is a `scipy.kvp` for computing the derivative, but that would require another Op without counterparts in different backends